### ISSUE TITLE
Update F5 configuration after remove and create Ingress resource

### ISSUE THREAD

**Issue by wheestermanstoyota** [F5Networks/k8s-bigip-ctlr#1093]
#### Description
Update F5 configuration after remove and create Ingress resource

#### Kubernetes Version
v1.14.7-docker-1 (Docker-EE 3.0)

#### Controller Version
1.11.1

#### Diagnostic Information

Ingress resource:

```yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: tme-ing-jetty-deploytest
  namespace: tme-ns-jetty-dev
  annotations:
    # See the k8s-bigip-ctlr documentation for information about
    # all Ingress Annotations
    # https://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/#supported-ingress-annotations
    virtual-server.f5.com/ip: "10.100.4.110"
    virtual-server.f5.com/partition: "KuberNetes"
    virtual-server.f5.com/balance: "round-robin"
    virtual-server.f5.com/http-port: "80"
    virtual-server.f5.com/health: |
      [
        {
          "type": "http",
          "path": "internal-docker-k8s-jetty-deploytest-dev.toyota-europe.com/",
          "send": "HEAD HTTP/1.0",
          "interval": 5,
          "timeout":  10
        }
      ]    
spec:
  rules:
  - host: internal-docker-k8s-jetty-deploytest-dev.toyota-europe.com
    http:
      paths:
      - path: /
        backend:
          serviceName: tme-svc-jetty-deploytest
          servicePort: 8080
```

We have two pods running (deployment + service). The we create the Ingress using kubectl create..., Inside the F5 console I see almost immediately the pool, but is red because no members inside. I takes around 2 minutes before the pool comes green with the members. I woudl expect that this should almost immediate because all pods were running. So why does it takes so much time?

Same behaviour when I remove the Ingress. I takes more then a minute before the pool is removed on F5 level.

Regards
Walter

**Comment by wheestermanstoyota** [comment]
Some additional info, we are using calico v3.8.2.

Calico process is running.

IPv4 BGP status
+----------------+-------------------+-------+----------+--------------------------------+
|  PEER ADDRESS  |     PEER TYPE     | STATE |  SINCE   |              INFO              |
+----------------+-------------------+-------+----------+--------------------------------+
| 10.100.114.7   | node-to-node mesh | up    | 00:00:45 | Established                    |
| 10.100.114.8   | node-to-node mesh | up    | 00:00:45 | Established                    |
| 10.100.114.9   | node-to-node mesh | up    | 10:10:37 | Established                    |
| 10.100.114.20  | node-to-node mesh | up    | 00:00:46 | Established                    |
| 10.100.114.21  | node-to-node mesh | up    | 00:00:46 | Established                    |
| 10.100.114.22  | node-to-node mesh | up    | 00:00:46 | Established                    |
| 10.100.114.23  | node-to-node mesh | up    | 00:00:46 | Established                    |
| 10.100.114.24  | node-to-node mesh | up    | 00:00:46 | Established                    |
| 10.100.114.25  | node-to-node mesh | up    | 00:00:46 | Established                    |
| 10.100.114.160 | global            | start | 00:00:44 | Active Socket: Host is         |
|                |                   |       |          | unreachable                    |
| 10.100.114.161 | global            | up    | 00:00:46 | Established                    |
| 10.100.114.162 | global            | up    | 00:00:46 | Established                    |
| 10.100.114.163 | global            | up    | 00:00:46 | Established                    |
+----------------+-------------------+-------+----------+--------------------------------+

10.100.114.160 is for the moment down, so that no issue.

**Comment by ssurenr** [comment]
@wheestermanstoyota We are working on a few performance issues in the controller when processing Ingress. We will consider this scenario.

**Comment by wheestermans31** [comment]
Thanks for the feedback, this will help us a lot if the pool members got available when the pool get created.

Regards
Walter

**Comment by walter-heestermans-toyota** [comment]
Any idea if this will be fixed in the newt release. If not, wjen can we expect?

Regards
Walter

**Comment by ssurenr** [comment]
1.13 is tracking for a January 2020 release.

**Comment by mdditt2000** [comment]
Issues resolved in CIS 1.13 with performance improvements
